package.json

```diff
@@ -13,8 +13,11 @@
         "serve:geoengine:nightly": "ng serve --configuration development geoengine-app --proxy-config proxy-geoengine.conf.json",
         "serve:data-atlas:local": "ng serve --configuration development data-atlas-app --proxy-config proxy-local.conf.json",
         "serve:data-atlas:nightly": "ng serve --configuration development data-atlas-app --proxy-config proxy-data-atlas.conf.json",
-        "build-prod:geoengine": "ng build --configuration production geoengine-app",
+        "serve:gfbio:local": "ng serve --configuration development gfbio-app --proxy-config proxy-local.conf.json",
+        "serve:gfbio:nightly": "ng serve --configuration development gfbio-app --proxy-config proxy-gfbio.conf.json",
```

Comment:
It is not part of _build-apps_.

Reply:
8513e422b354db49006b98e63a015f0a4639fbe5